bolt/src/main/java/com/slack/api/bolt/App.java

```diff
@@ -490,6 +490,8 @@ public App(AppConfig appConfig, Slack slack, List<Middleware> middlewareList) {
         this.oAuthExceptionHandler = new OAuthDefaultExceptionHandler(config());
 
         this.oAuthCallbackService = null; // will be initialized by initOAuthServicesIfNecessary()
+
+        this.status = Status.Stopped;
```

Comment:
@seratch NPE from `synchronized(status)` blocks is thrown if status is not set in the controller. Do you want me to create a different PR for this change?